### PR TITLE
fix(cli): include full eleventy config

### PR DESCRIPTION
**Pull request by jorgecasar**
## What I did

1. Create an object with all eleventyConfig method as empty functions

### CODE REVIEW

File: packages/cli/src/normalizeConfig.js [hunk 1]
```diff
@@ -38,23 +39,7 @@ export async function normalizeConfig(inConfig) {
...
-          addHandlebarsShortcode: () => {},
-          addShortcode: () => {},
-          addLiquidTag: () => {},
-          addNunjucksTag: () => {},
-          setTemplateFormats: () => {},
-          setQuietMode: () => {},
-          addTransform: () => {},
-          addLinter: () => {},
-        });
+        newEleventyConfig = fileConfig.eleventy(eleventyConfig);
```
Comment (daKmoR) [inline packages/cli/src/normalizeConfig.js:42]: I'm not sure if that would have side effects 🤔

adding `addPlugin: () => {},` might be saver 🤔

Comment (jorgecasar) [reply packages/cli/src/normalizeConfig.js:42]: Ok, I will check all functions we could need and include at once

Comment (jorgecasar) [reply packages/cli/src/normalizeConfig.js]: I created an object with all empty functions that I guess could be needed

Comment (daKmoR) [reply packages/cli/src/normalizeConfig.js]: nice 👍

**Comment by daKmoR** [comment]
thank you 🤗

**Pull request merged**